cli/src/pcluster/config/cluster_config.py

```diff
@@ -614,6 +615,15 @@ def existing_dns_name(self):
         ]["Nfs"]["DNSName"]
 
 
+class ExistingFsxFileCache(BaseSharedFsx):
```

Comment:
*Observation to address in a separate PR, but worth starting the discussion for it.*

The file `cluster_config.py` is getting bigger and bigger and this is not a good practice.
We should consider to refactor it and put classes definition in specific files.

For example, we should move every classes related to the shared storage into a dedicated `shared_storage_config.py` file.

Let's think about it as a follow-up improvement.